apps/ledger-live-mobile/src/hooks/useFloorPrice.ts

```diff
@@ -0,0 +1,55 @@
+import { useEffect, useMemo, useState } from "react";
+import { CryptoCurrency } from "@ledgerhq/types-cryptoassets";
+import { BigNumber } from "bignumber.js";
+import { getFloorPrice } from "@ledgerhq/live-common/nft/index";
+import { FloorPrice, ProtoNFT } from "@ledgerhq/types-live";
+import {
+  findCryptoCurrencyByTicker,
+  getCryptoCurrencyById,
+  valueFromUnit,
+} from "@ledgerhq/coin-framework/currencies/index";
+
+const useFloorPrice = (nft: ProtoNFT) => {
```

Comment:
NOTE: The content of this hook was simply taken from somewhere else. I've moved it here so I can use it in multiple places.